arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+					"type": "string"
+				}, {
+					"name": "importDevicesRequest",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"$ref": "#/definitions/ImportDevicesRequest"
+					}
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/JobResponse"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		}
+	},
+	"definitions": {
+		"Object": {
+			"type": "object",
+			"properties": {}
+		},
+		"SharedAccessSignatureAuthorizationRule": {
+			"type": "object",
+			"properties": {
+				"keyName": {
+					"type": "string"
+				},
+				"primaryKey": {
+					"type": "string"
+				},
+				"secondaryKey": {
+					"type": "string"
+				},
+				"rights": {
+					"enum": ["RegistryRead", "RegistryWrite", "ServiceConnect", "DeviceConnect"],
+					"type": "string"
+				}
+			}
+		},
+		"IotHubProperties": {
+			"type": "object",
+			"properties": {
+				"authorizationPolicies": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+					}
+				},
+				"hostName": {
+					"type": "string"
+				},
+				"eventHubEndpoints": {
+					"type": "object",
+					"additionalProperties": {
+						"$ref": "#/definitions/EventHubProperties"
+					}
+				},
+				"storageEndpoints": {
+					"type": "object",
+					"additionalProperties": {
+						"$ref": "#/definitions/StorageEndpointProperties"
+					}
+				},
+				"messagingEndpoints": {
+					"type": "object",
+					"additionalProperties": {
+						"$ref": "#/definitions/MessagingEndpointProperties"
+					}
+				},
+				"enableFileUploadNotifications": {
+					"type": "boolean"
+				},
+				"cloudToDevice": {
+					"$ref": "#/definitions/CloudToDeviceProperties"
+				},
+				"comments": {
+					"type": "string"
+				},
+				"operationsMonitoringProperties": {
+					"$ref": "#/definitions/OperationsMonitoringProperties"
+				},
+				"features": {
+					"enum": ["None", "DeviceManagement"],
+					"type": "string"
+				}
+			}
+		},
+		"IotHubSkuInfo": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"enum": ["F1", "S1", "S2"],
+					"type": "string"
+				},
+				"tier": {
+					"enum": ["Free", "Standard"],
+					"type": "string",
+					"readOnly": true
+				},
+				"capacity": {
+					"format": "int64",
+					"type": "integer"
+				}
+			}
+		},
+		"EventHubProperties": {
+			"type": "object",
+			"properties": {
+				"retentionTimeInDays": {
+					"format": "int64",
+					"type": "integer"
+				},
+				"partitionCount": {
+					"format": "int32",
```

Comment:
please provide description every where. If the service code does not have documentation then add it, so that swashbuckle can put it in the swagger doc. This will be beneficial to your team and also for customers